File: Haz.py
import math
n, mm = input().split()
n = int(n)
mm = int(mm)
y = [int(y) for y in input().split()]
 
if n == len(y):

    avg = sum(y)/n
    if math.modf(avg)[0] == 0.5:
        avg += 0.01
        avg = round(avg)
        print(n*(mm - avg))
    else :
        avg = round(avg)
        print(n*(mm - avg))

File: haz.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() 
{
    int n, k; 
    cin >> n >> k;
    int arr[n];
    int sum = 0;
    for(int i = 0; i < n; i++)
    {
        cin >> arr[i];
        sum += arr[i];
    }

    int avg = 0;

    for(int i = 0; i < k*n; ++i)
    {
        avg = round((sum/n) + i/n);
        if (avg == k)
        {
            cout << i;
            break;
        }
    }
}
